factcast-store/src/main/java/org/factcast/store/internal/PgFactStore.java

```diff
@@ -43,7 +41,6 @@
 import org.factcast.store.internal.lock.FactTableWriteLock;
 import org.factcast.store.internal.query.PgFactIdToSerialMapper;
 import org.factcast.store.internal.query.PgQueryBuilder;
-import org.factcast.store.internal.snapcache.SnapshotCache;
 import org.factcast.store.registry.SchemaRegistry;
 import org.jetbrains.annotations.NotNull;
```

Comment:
Do we have a ticket for the getting rid of the table in 0.8.1 or something like this?

Reply:
How do we handle the protocol level snapshot support? 🤔

Reply:
To be done after 0.8.0 :D 
https://github.com/factcast/factcast/issues/3033

Reply:
Now we have a ticket for getting rid of the table :D
https://github.com/factcast/factcast/issues/3033